src/frequenz/channels/util/_merge.py

```diff
@@ -20,6 +20,12 @@ class Merge(Receiver[T]):
         stream, by using `Merge` like this:
 
         ```python
+        from frequenz.channels import Broadcast
+
+        chan = Broadcast[int]("input-chan")
+        receiver1 = chan.new_receiver()
+        receiver2 = chan.new_receiver()
+
         merge = Merge(receiver1, receiver2)
```

Comment:
we should probably make two channels and merge their receivers,  else it gets a bit confusing I think.